.github/workflows/tests.yml

```diff
@@ -205,6 +196,15 @@ jobs:
           path: ${{env.GOCOVERDIR}}
         if: env.GOCOVERDIR != ''
 
+      - name: Strip binaries/libs
```

Comment:
why is this ordering change needed?

Reply:
Seems like doing it this late is what made them all be stripped in the uploaded `.zip` that goes from ~145 MiB to 107MiB.

Reply:
weird, are they being built twice or something?

Reply:
That's what I suspected but couldn't track down.

Reply:
Do we need to not strip them for the tics run?